### ISSUE TITLE
how to put option  trace: true, in here?

### ISSUE THREAD

**Issue by shipcake** [CharlesStover/reactn-devtools#7]
hi how to put composeWithDevTools({trace: true,})

thank.

**Comment by CharlesStover** [comment]
This is not currently supported, but a great feature request.

It looks like the change would be to accept another parameter for dev tools options and pass them on [line 75](https://github.com/CharlesStover/reactn-devtools/blob/master/src/index.ts#L75) to `compose`.

I think I like something like this to maintain non-breaking backwards compatibility and ease of use:
```javascript
declare function addReactNDevTools(
  options?: Options
);
declare function addReactNDevTools(
  Provider: ReactNProvider,
  options?: Options,
);
function addReactNDevTools(
  Provider?: ReactNProvider | Options
  options?: Options
) { ... }
```

I'll let it stew for a little bit before implementing this.

**Comment by CharlesStover** [comment]
Thank you for this great suggestion. Sorry for the delay. It should be released in 1.1.0 later today.
